src/error/specification

```diff
@@ -310,10 +310,11 @@ ingroup:plugin
 module:network
 
 number:52
-description:could not type check value of key
+description:error in the type plugin
```

Comment:
Isn't "error in the type plugin" way too generic? What about "the value does not match with the type of the key"?

Reply:
The description is going to be removed with the error concept anyways (hopefully soon). The reasons used in code are much more descriptive.

Reply:
As long as we have the old error system we need to live with what we have. Hopefully @Piankero finishes soon!